src/apps/ojph_compress/ojph_compress.cpp

```diff
@@ -461,9 +461,9 @@ int main(int argc, char * argv[]) {
 
     "The following option has a default value (optional):\n"
     " -num_decomps  (5) number of decompositions\n"
-    " -qstep        (1.0/255,0) quantization step size; all quantization\n"
-    "               step sizes are derived from this.\n"
-    " -reversible   (false) for irreversible adnd true for reversible\n"
+    " -qstep        (-1.0...1.0) quantization step size; all quantization without 0.0\n"
```

Comment:
-qstep      (0.0039 for 8-bit data) quantization step size; quantization step sizes for all subbands are derived from this value. qstep value should be from a very small positive float number such as 0.00001 and up to 0.5.  Larger values produce smaller files (i.e., coarsely quantization).

I think there is no need to mention the -1.0; it is just an implementation issue that I might change,